### PR TITLE
Minor tweaks for consistency. Fix some typos

### PR DESCRIPTION
**Pull request by robfallows**
Numbered/bulleted lists items were mainly terminated with a "." - ensured this on all of them.
Clarified the statement around MongoDB joins (3.2 does introduce a sort of left outer join).
Found/fixed a few more typos.
<a href='#crh-start'></a><a href='#crh-data-%7B%22processed%22%3A%20%5B%22https%3A//github.com/meteor/guide/pull/117%23discussion_r46553888%22%2C%20%22https%3A//github.com/meteor/guide/pull/117%23discussion_r46639049%22%5D%2C%20%22comments%22%3A%20%7B%22Pull%206d43a9e2b622d19fd1603cfff6aff7297f41582c%20content/collections.md%2027%22%3A%20%7B%22html_url%22%3A%20%22https%3A//github.com/meteor/guide/pull/117%23discussion_r46553888%22%2C%20%22comments%22%3A%20%5B%7B%22body%22%3A%20%22This%20does%20now%20appear%20to%20be%20available.%22%2C%20%22created_at%22%3A%20%222015-12-03T14%3A04%3A18Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars.githubusercontent.com/u/6086035%3Fv%3D3%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/robfallows%22%7D%7D%2C%20%7B%22body%22%3A%20%22%3A%2B1%3A%20%22%2C%20%22created_at%22%3A%20%222015-12-04T01%3A36%3A43Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars.githubusercontent.com/u/132554%3Fv%3D3%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/tmeasday%22%7D%7D%5D%2C%20%22title%22%3A%20%22File%3A%20content/collections.md%3AL127-140%22%7D%7D%7D'></a>
<a href='https://www.codereviewhub.com/'></a>
- [x] <a href='#crh-comment-Pull 6d43a9e2b622d19fd1603cfff6aff7297f41582c content/collections.md 27'></a> &nbsp;<b><a href='https://github.com/meteor/guide/pull/117#discussion_r46553888'>File: content/collections.md:L127-140</a></b>
- <a href='https://github.com/robfallows'></a> This does now appear to be available.

<a href='https://www.codereviewhub.com/meteor/guide/pull/117?mark_as_completed=1'></a>&nbsp;<a href='https://www.codereviewhub.com/meteor/guide/pull/117?approve=1'></a>&nbsp;<a href='https://github.com/meteor/guide/pull/117'></a>
<a href='#crh-end'></a>

### CODE REVIEW

**Comment by stubailo** [comment]
This pull request is now managed with CodeReviewHub. <a href='https://github.com/meteor/guide/pull/117'>Refresh</a>.<a href='#crh-update-none'></a>

File: content/collections.md [hunk 1]
```diff
@@ -127,15 +127,13 @@ const list = {
 Lists.schema.validate(list);
 ```
 
-// XXX: this isn't actually the case yet. We are waiting on https://github.com/meteor/method/issues/4
```
Comment (robfallows) [inline content/collections.md]: This does now appear to be available.

Comment (tmeasday) [inline content/collections.md]: :+1:

**Comment by tmeasday** [comment]
Thanks again Rob!

**Pull request merged**